File: client/src/state/model/write/Scene.ts
import type {Key} from "../Key";

export interface Scene {
  name: string;
  urlname: string;
  key: string;

  raw: string;
  tags: Array<string>;
  category: Array<string>;

  outLinks: Array<Key>;
  inLinks: Array<Key>;
  qualities: Array<Key>;
}

File: service/GoodNight.Service.Domain/Model/Write/Transfer/Scene.cs
using System.Collections.Generic;

namespace GoodNight.Service.Domain.Model.Write.Transfer
{
  public record Scene(
    string Name,
    string Story,

    string Raw,

    IEnumerable<Reference> OutLinks,
    IEnumerable<Reference> InLinks,
    IEnumerable<Reference> Qualities);
}
